src/iOS.Core/Controllers/BaseLockPasswordViewController.cs

```diff
@@ -367,10 +367,21 @@ public async Task PromptBiometricAsync()
             await _stateService.SetBiometricLockedAsync(!success);
             if (success)
             {
-                DoContinue();
+                var userKey = await _cryptoService.GetBiometricUnlockKeyAsync();
+                await SetUserKeyAndContinueAsync(userKey);
             }
         }
 
+        private async Task SetUserKeyAndContinueAsync(UserKey key)
```

Comment:
There's a `SetKeyAndContinue` on line 403 we could use instead of creating a new one.

Reply:
You are right!